cfme/physical/provider/__init__.py

```diff
@@ -37,6 +38,14 @@ def __init__(
         self.name = name
         self.key = key
 
+    @property
+    def hostname(self):
+        return getattr(self.default_endpoint, "hostname", None)
+
+    @property
```

Comment:
**REQUIRED:** I'm not sure I like this, it means do a DNS resolution during collection which at the moment could be considered suboptimal.

Reply:
Do you mean use **host name (ex: integration.tests)** instead an **IP address (ex: 10.0.0.1)**? @psav